### PR TITLE
feat: Serializable execution details

### PR DESCRIPTION
**Pull request by anticorrelator**
- Adds example notebook that shows how to deal with errors using phoenix evals
- Returns strings instead of objects for execution status and exception info when setting `include_exceptions=True`

### CODE REVIEW

**Review by mikeldking — APPROVED** [review]

File: tutorials/evals/evaluations_with_error_handling.ipynb [hunk 1]
```diff
@@ -0,0 +1,1638 @@
+{
```
Comment (axiomofjoy) [inline tutorials/evals/evaluations_with_error_handling.ipynb:1]: <p>Run relevance evals</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/3358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (axiomofjoy) [inline tutorials/evals/evaluations_with_error_handling.ipynb:1]: <p>if input data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/3358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (axiomofjoy) [inline tutorials/evals/evaluations_with_error_handling.ipynb:1]: <p>Suggestion: Use <code>.value_counts()</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/3358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (axiomofjoy) [inline tutorials/evals/evaluations_with_error_handling.ipynb:1]: <p>Same as above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/3358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

**Review by axiomofjoy — APPROVED** [review]
Nice, maybe link the notebook from the docs?

**Pull request merged**